docs/source/index.rst

```diff
@@ -14,7 +14,16 @@ transformers in this library all share a few useful properties:
  * Can explicitly configure which columns in the data are encoded by name or index, or infer non-numeric columns regardless of input type
  * Can drop any columns with very low variance based on training set optionally
  * Portability: train a transformer on data, pickle it, reuse it later and get the same thing out.
- * Full compatibility with sklearn pipelines, input an array-like dataset like any other transformer
+ * Full compatibility with sklearn pipelines, input an array-like dataset like any other transformer (\*)
+
+(\*) For full compatibility with Pipelines and ColumnTransformers, and consistent behaviour of `get_feature_names_out`, it's recommended to upgrade `sklearn` to a version at least '1.2.0' and to set output as pandas:
```

Comment:
could you please introduce a section `known issues` here after usage and before contents, that states something like:

"""
CategoryEncoders internally works with pandas `DataFrames` as apposed to sklearn which works with numpy arrays. This can cause problems in sklearn versions prior to `1.2.0`. In order to ensure full compatibility with sklearn set sklearn to also output `DataFrames`. This can be done by
```python
sklearn.set_config(transform_output="pandas")
```
for a whole project or just for a single pipeline using
```python
Pipeline(
    steps=[
        ("preprocessor", SomePreprocessor().set_output("pandas"),
        ("encoder", SomeEncoder()),
    ]
)
```
If you experience another bug feel free to report it on github https://github.com/scikit-learn-contrib/category_encoders/issues

"""
I think this makes it sufficiently clear and is better than the status quo. Do you agree?

Reply:
I think it's very good, yes. Thanks! I don't see any need for more. Let me add this piece and remove the other one so that you can merge :)